csharp/BUILDING.md

```diff
@@ -131,8 +120,13 @@ To create a NuGet package, open a command prompt and run the following command:
 dotnet msbuild msbuild/ice.proj /t:NuGetPack
 ```
 
-This creates the `zeroc.ice.net` Nuget package in the `msbuild/zeroc.ice.net`
-directory.
+This creates the `zeroc.ice.net` Nuget package in the `msbuild/zeroc.ice.net` directory.
+
+You can publish the package to your local `global-packages` source with the following command:
+
+```shell
+dotnet msbuild msbuild\ice.proj /t:Publish
```

Comment:
For a follow-up PR: is there anyway we can switch to "dotnet build" and not need the `msbuild/ice.proj` argument?

Reply:
We need MSBuild for the C++ dependencies. If we build then separately it would be fine to use dotnet.